language/move-lang/src/expansion/ast.rs

```diff
@@ -256,7 +261,8 @@ pub enum Exp_ {
     While(Box<Exp>, Box<Exp>),
     Loop(Box<Exp>),
     Block(Sequence),
-    Lambda(LValueList, Box<Exp>), // spec only
+    Lambda(LValueList, Box<Exp>),                    // spec only
+    Quant(QuantKind, LValueWithRangeList, Box<Exp>), // spec only
```

Comment:
Wondering whether we should right away add triggers here. That would be another Vec<Exp>. I think we want to have the power to do triggers manually, though hopefully, in most cases the user  can rely on some automatic generation.

Reply:
I wanted to discuss some options with you all, now that I understand the details a bit more.
I also need to finish parsing/translating quantifications over multiple variables.

So it seemed reasonable to review/land this chunk of commits and open new PR(s).